sound/soc/sof/intel/mtl.c

```diff
@@ -692,6 +692,9 @@ int sof_mtl_ops_init(struct snd_sof_dev *sdev)
 
 	sof_mtl_ops.pcm_pointer = mtl_dsp_pcm_pointer;
 
+	sof_mtl_ops.hw_info |= SNDRV_PCM_INFO_HAS_LINK_ATIME |
+				SNDRV_PCM_INFO_HAS_WALL_CLOCK;
```

Comment:
If I want to be picky, this is 3 patch:
ASoC: SOF: pcm: Add support for get_time_info callback
ASoC: SOF: ipc4-pcm: Implement get_time_info
ASoC: SOF: Intel: mtl: Enable LINK_ATIME and WALL_CLOCK support

But at least: `s/Asoc/ASoC` and `SOF: ipc4/mtl:`

Reply:
sure, thanks